test/Nancy.Tests/project.json

```diff
@@ -1,6 +1,8 @@
 {
+    "version": "1.4.1.0",

```

Comment:
Understood. The removal was a small pain point for us over at FakeItEasy too.
I thought I'd heard that binary serialization was coming back sometime, but I cannot find a reference now.